### ISSUE TITLE
ValueError: 'size' must be a 1-D int32 Tensor

### ISSUE THREAD

**Issue by vardanagarwal** [de-code/python-tf-bodypix#44]
Getting a Value error on some images while not in some. 

Entire Traceback:

```
Traceback (most recent call last):
  File "main.py", line 297, in <module>
    res = run_face_id(face_obj, img)
  File "main.py", line 28, in run_face_id
    text5 = face_obj.check_background()
  File "main.py", line 219, in check_background
    result = self.bodypix_model.predict_single(self.img)
  File "/home/vardan/face_id/lib/python3.7/site-packages/tf_bodypix/model.py", line 293, in predict_single
    model_input_image, padding = self.get_padded_and_resized(image, model_input_size)
  File "/home/vardan/face_id/lib/python3.7/site-packages/tf_bodypix/model.py", line 287, in get_padded_and_resized
    model_input_size.width
  File "/home/vardan/face_id/lib/python3.7/site-packages/tf_bodypix/bodypix_js_utils/util.py", line 79, in pad_and_resize_to
    resized = tf.image.resize([padded], [target_height, target_width])[0]
  File "/home/vardan/face_id/lib/python3.7/site-packages/tensorflow/python/util/dispatch.py", line 201, in wrapper
    return target(*args, **kwargs)
  File "/home/vardan/face_id/lib/python3.7/site-packages/tensorflow/python/ops/image_ops_impl.py", line 1546, in resize_images_v2
    skip_resize_if_same=False)
  File "/home/vardan/face_id/lib/python3.7/site-packages/tensorflow/python/ops/image_ops_impl.py", line 1226, in _resize_images_common
    raise ValueError('\'size\' must be a 1-D int32 Tensor')
ValueError: 'size' must be a 1-D int32 Tensor
```

The size of one of the images in question was (145, 226, 3) by width, height, and channels. Is there a minimum size requirement that may be causing it?

**Comment by de-code** [comment]
Hi, thanks for raising the issue.

Would you be able to share the code you are using and the image causing the issue?

Thank you

**Comment by vardanagarwal** [comment]
The code:
```
from tf_bodypix.api import download_model, load_model, BodyPixModelPaths
import cv2
import numpy as np

bodypix_model = load_model(download_model(
            BodyPixModelPaths.MOBILENET_FLOAT_100_STRIDE_16))
img = cv2.imread('img.jpg')
result = bodypix_model.predict_single(img)
body_mask = result.get_mask(threshold=0.25).numpy()
body_mask = (body_mask*255).astype(dtype=np.uint8)
```

You can get the image from here: https://drive.google.com/file/d/1hXss_DPnjUxTXrbGnEDaIk4JS6QUukpo/view?usp=sharing

**Comment by vardanagarwal** [comment]
Thanks for the real fast work. Could you please clarify if I should I just update the pip installed library or is there is any changes needed to be done in the code.

**Comment by de-code** [comment]
You are fast too. I was just going to update you. I also released a new version 0.3.2 which will include the fix (might take a moment before it's properly available and not cached).

**Comment by vardanagarwal** [comment]
Sure thanks
